docs/theming/developers.md

```diff
@@ -200,46 +207,47 @@ themable graphics
 
 ## How theming works
 
-We developed our theming system with web standards in mind, aiming for it to use
-CSS over JavaScript as much as possible. At the current state of the art, we
-found that some JavaScript is required for the system to work correctly, but in 
-the near  future, we expect to be able to remove all or nearly all of that 
-JavaScript, and have the theming system work entirely (or almost entirely) 
+We developed our theming system with web standards in mind, aiming to use
+CSS over JavaScript as much as possible. At the current state of browser technology,
+we found that some JavaScript is required for the system to work correctly.
```

Comment:
`we` → `we've`

Reply:
[917e417](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/917e4175b7f4667de89166632a801d9179979bd1)